### PR TITLE
Fix for Ipad potrait devices

### PR DESCRIPTION
**Pull request by nerdlet**
## Description

#168216289

## Type of change
- [X] Bug fix (non-breaking change which fixes an issue)

### Fixes to Profile page
Screenshot 2019-09-06 at 18 21 18
Screenshot 2019-09-06 at 18 21 10
Screenshot 2019-09-06 at 18 21 26
Screenshot 2019-09-06 at 18 21 34
Screenshot 2019-09-06 at 18 21 40

###Fixes to other broken components/pages

Screenshot 2019-09-09 at 15 44 25
Screenshot 2019-09-09 at 15 44 06
Screenshot 2019-09-09 at 15 44 58
Screenshot 2019-09-09 at 15 45 11
<img width=
screenshot
Screenshot 2019-09-09 at 15 43 44
Screenshot 2019-09-09 at 15 43 33

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by kilemensi** [comment]
Something to think about @nerdlet @KhadijaMahanga @karimkawambwa: Shouldn't we just customize the breakpoint values: https://material-ui.com/customization/default-theme/?expend-path=$.breakpoints.values to include iPad Portrait into `md`?

This way, we won't have to do deal with custom media queries altogether. `md` and up would still mean `desktop`/`full` view.

**Comment by karimkawambwa** [comment]
> Something to think about @nerdlet @KhadijaMahanga @karimkawambwa: Shouldn't we just customize the breakpoint values: https://material-ui.com/customization/default-theme/?expend-path=$.breakpoints.values to include iPad Portrait into `md`?
> 
> This way, we won't have to do deal with custom media queries altogether. `md` and up would still mean `desktop`/`full` view.

This is a cool feature to note going forward with material-ui 👍 @kilemensi

**Comment by nerdlet** [comment]
@kilemensi Done. Created a story to review site on medium devices for all other pages/components

File: src/components/CountryPartners/index.js [hunk 1]
```diff
@@ -20,9 +20,11 @@ const styles = theme => ({
   },
   layout: {
     padding: '1rem',
+    alignItems: 'flex-start',
     [theme.breakpoints.up('md')]: {
       maxWidth: '81.3571429rem',
-      margin: '0 auto'
+      margin: '0 auto',
+      alignItems: 'flex-start'
```
Comment (kilemensi) [inline src/components/CountryPartners/index.js]: No need to repeat if it's same as above

File: src/components/HowItWorks/Map.js [hunk 2]
```diff
@@ -5,6 +5,8 @@ import { withStyles } from '@material-ui/core/styles';
 
 import background from '../../assets/images/hero-image-3_2.png';
 
+// Use @media queries for ipad portait devices: 768px
+// eslint-disable-line no-useless-computed-key
```
Comment (kilemensi) [inline src/components/HowItWorks/Map.js]: I don't see the need for the comment or for disabling the eslint here.

File: src/components/HowItWorks/index.js [hunk 3]
```diff
@@ -39,8 +39,11 @@ const styles = theme => ({
   },
   description: {
     [theme.breakpoints.up('md')]: {
-      paddingTop: '5.286rem', // 74px / 16
-      paddingLeft: '5.286rem' // 74px / 16
+      paddingLeft: '3.286rem', // 74px / 16
```
Comment (kilemensi) [inline src/components/HowItWorks/index.js]: Remove/Update the comment if you change the code: `74/16 != 3.3`

**Comment by nerdlet** [comment]
@kilemensi Done with the update. Please let me know if there is something I have missen here

**Comment by nerdlet** [comment]
@kilemensi Ping

File: src/components/Hero/CountryHero.js [hunk 4]
```diff
@@ -20,23 +20,23 @@ const styles = theme => ({
   root: {
     flexGrow: 1,
     marginTop: '32px',
-    [theme.breakpoints.up('md')]: {
+    [theme.breakpoints.up('lg')]: {
```
Comment (kilemensi) [inline src/components/Hero/CountryHero.js]: Don't switch `md` to `lg`, this will make `md` look _mobile_ and not _desktop_ (which is what we want).

**Pull request closed without merging**